allennlp/models/semantic_role_labeler.py

```diff
@@ -209,13 +214,18 @@ def from_params(cls, vocab: Vocabulary, params: Params) -> 'SemanticRoleLabeler'
         text_field_embedder = TextFieldEmbedder.from_params(vocab, embedder_params)
         stacked_encoder = Seq2SeqEncoder.from_params(params.pop("stacked_encoder"))
         binary_feature_dim = params.pop("binary_feature_dim")
-        initializer = InitializerApplicator.from_params(params.pop("initializer", []))
```

Comment:
Can we leave the initializer being initialized to empty rather than it being `None`? This way, it will log the default initializations of all the modules, which is important so you can notice if they are inappropriate.

Reply:
This can be in the constructor actually : `initializer: InitializerApplicator = InitializerApplicator(None)`

Reply:
done